docs/cmd/completion/script.md

```diff
@@ -1,11 +1,16 @@
-# Driftctl completion script
+---
+id: script
+title: Completion Script
```

Comment:
> Ok so markdown cannot be viewed from both doc website and github (broken images, missing titles)

Not exactly, broken images and missing titles can be taken care of. The front matter here has different attributes to make it work as we want. BUT docusaurus and more generally mdx files need front matter to make it work as expected. So it comes down to whether it is bad or not to have the front matter in the first 4-5 lines of the md file displayed.

> We should be clear about it and we need to find a true separation between developer documentation (we want to keep this one in the repo) and user doc (website)

> Maybe create a folder website/ at the root of repo with all markdown for the website, and keep a doc/ folder for developer related stuff.

As I said in the first comment, if we keep all our docs in the `doc` or `docs` (I can rename it, I just prefer to be plurals), Docusuraus, in their implementation of versioning, will copy EVERYTHING to the `versioned_docs` folder, whether it is used or not by the documentation website.

Reply:
> Not exactly, broken images and missing titles can be taken care of. The front matter here has different attributes to make it work as we want. BUT docusaurus and more generally mdx files need front matter to make it work as expected. So it comes down to whether it is bad or not to have the front matter in the first 4-5 lines of the md file displayed.

How does it looks on docusaurus when we put a level 1 title ? I think it could be acceptable to have the front matter before the title.

But as I said in a our demo call, user documentation files will no longer be made to be read on github anymore, so maybe we could ignore theses remarks.

Maybe `doc` (or `docs` I dont care) should be used for documentation that we want to be viewed using github (contributing and technical stuff) and everything else (user documentation) could be in `website` folder ?

More generally talking, I don't want our user to land on the repo, click on the doc folder, and find broken markdown files. It should be clear that documentation should be read on `doc.driftctl.com`. And thus if we have a `docs` folder it could be confusing.